### PR TITLE
Dev: Update Cygwin install instructions

### PR DESCRIPTION
**Pull request by stephendade**
Updating the Cygwin install to Python3. Based on updated install script over at https://github.com/ArduPilot/ardupilot/pull/14095

### CODE REVIEW

**Comment by Hwurzburg** [comment]
LGTM

File: dev/source/docs/building-setup-windows-cygwin.rst [hunk 1]
```diff
@@ -123,11 +123,13 @@ Set up directories/paths and extra packages in Cygwin
 
 #. open and then close the "Cygwin64 Terminal" application from the desktop or start menu icon.  This creates initialisation files for the user in the Cygwin home directory.
 
-#. open the "Cygwin64 Terminal" application from the desktop or start menu icon.  Enter the following command to install additional Python packages:
+#. open the "Cygwin64 Terminal" application from the desktop or start menu icon.  Enter the following command to set Python 3.6 to be the default Python and then install additional Python packages:
 
 ::
 
-    pip2 install empy pyserial pymavlink
+    ln -s /usr/bin/python3.6 /usr/bin/python
```
Comment (khancyr) [inline dev/source/docs/building-setup-windows-cygwin.rst:130]: doesn't python-is-python3 or python-is-python2 exist in cygwin ?

Comment (stephendade) [reply dev/source/docs/building-setup-windows-cygwin.rst:130]: > doesn't python-is-python3 or python-is-python2 exist in cygwin ?

Correct. It previously was python-is-python2, but Cygwin has removed that from their installer a few months ago.

**Comment by stephendade** [comment]
Note that there's regular posts on the Ardupilot forums from new Cygwin users (ie https://discuss.ardupilot.org/t/build-problem-in-cygwin-usr-bin-env-python/54915) encoutnering this issue. So it would be good to get this merged soon!

File: dev/source/docs/building-setup-windows-cygwin.rst (hunk 1 above)
Comment (khancyr) [reply dev/source/docs/building-setup-windows-cygwin.rst:130]: oh ok ! I just found it on ubuntu20.04 and I found it much simplier that dealing with symlink, that is why I ask if the same thing exist on cygwin.

**Pull request merged**